pontoon/base/templates/translate.html

```diff
@@ -326,7 +326,13 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
           <button id="clear" title="Clear Translation (Ctrl + Shift + Backspace)">Clear</button>
           <button id="save" title="Save Translation (Enter)"></button>
         {% else %}
-          <p id="sign-in-required"><a id="sidebar-signin" href="{{ provider_login_url(request) }}">Sign in</a> to translate.</p>
+          <p id="sign-in-required">
```

Comment:
Its not working currently, as main.js adds next parameter to `#sidebar-signin`. Already asked @mathjazz in irc about it. Possible solutions:
1. Adding custom id for standalone url (will require adding css rule (only color)
2. Adding support for next parameter in standalone login